data/core.yaml

```diff
@@ -830,6 +830,7 @@ en:
         alreadyFixed: Already Fixed
         notAnIssue: Not an Issue
         login: You must log in to change Maproulette challenge.
```

Comment:
For consistency (and because that's what they call themselves) we should use capital-R  "MapRoulette" in strings, not "Maproulette".